From 1c7449928fd21943ae9526e05abb5f4d1db7a2ae Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Sun, 26 Feb 2017 17:41:35 +0200
Subject: [PATCH 2/5] Undefine __USE_MINGW_ANSI_STDIO as otherwise stdio.h
 defines asprintf

And configure did not find it because it's an inline function in stdio.h,
not a function that is available in some library. Fixes compilation on Windows.
---
 gl/asprintf.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gl/asprintf.c b/gl/asprintf.c
index 6cbd062..76f4acf 100644
--- a/gl/asprintf.c
+++ b/gl/asprintf.c
@@ -21,6 +21,9 @@
 #ifdef IN_LIBASPRINTF
 # include "vasprintf.h"
 #else
+# ifdef __USE_MINGW_ANSI_STDIO
+#  undef __USE_MINGW_ANSI_STDIO
+# endif
 # include <stdio.h>
 #endif
 
-- 
2.23.0.rc1

